WooCommerce/Classes/ViewRelated/Orders/Order Details/AddOns/OrderAddOnListViewModel.swift

```diff
@@ -44,9 +44,15 @@ final class OrderAddOnListI1ViewModel: ObservableObject {
     /// The `attribute.name` comes in the form of "add-on-title (add-on-price)". EG: "Topping (Spicy) ($30.00)"
     ///
     private static func addOnName(from attribute: OrderItemAttribute) -> String {
-        attribute.name.components(separatedBy: " (") // "Topping (Spicy) ($30.00)" -> ["Topping", "Spicy)", "$30.00)"]
-            .dropLast()                              // ["Topping", "Spicy)", "$30.00)"] -> ["Topping", "Spicy)"]
-            .joined(separator: " (")                 // ["Topping", "Spicy)"] -> "Topping (Spicy)"
+        let components = attribute.name.components(separatedBy: " (") // "Topping (Spicy) ($30.00)" -> ["Topping", "Spicy)", "$30.00)"]
```

Comment:
Since this looks like an important logic, should we consider merging this with `AddOnCrossreferenceUseCase.extractAddOnName` somehow? Not sure if it's worth it if we're going to replace this later on anyway. 🤷

Reply:
Yeah, I expect all of this to be temporary. Also, the code here, while similar, is meant to be only for displaying/cosmetic purposes so I don't want to mix it with the cross-reference one!